src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockImporter.scala

```diff
@@ -331,7 +334,8 @@ object BlockImporter {
       ommersPool: ActorRef,
       broadcaster: ActorRef,
       pendingTransactionsManager: ActorRef,
-      supervisor: ActorRef
+      supervisor: ActorRef,
+      node: BlockchainConfigBuilder
```

Comment:
_Why is this variable called "node" ?
On first read, I would have called it according to its function (like blockchainConfigBuilder) instead of its state (I recal lthat it's the node that provides the config retrieval feature)._

redacted since it relates to a later comment by @AurelienRichez